README.rst

```diff
@@ -1,7 +1,9 @@
-##############
+==============
 analysis_tools
-##############
+==============
 
-``analysis_tools`` is a package in the `LSST Science Pipelines <https://pipelines.lsst.io>`_.
+The Analysis Tools package is designed to assist in the creation of quality assurance (QA) plots and metrics from the outputs of an `LSST data reduction pipeline <https://pipelines.lsst.io/modules/lsst.pipe.base/creating-a-pipeline.html>`_.
+The intention is that users have the flexibility to construct complex data analysis tasks from a set of simple building blocks.
+These analysis tools therefore allow users to generate a series of consistent, repeatable and high-quality plots and metrics, regardless of which dataset they are run on.
```

Comment:
I don't think the "therefore" is necessary.

```suggestion
Analysis tools allows users to generate a series of consistent, repeatable and high-quality plots and metrics, regardless of which dataset they are run on.
```

Reply:
It helps if you read the fully rendered paragraph, I find: https://github.com/lsst/analysis_tools/tree/tickets/DM-36208.

If sentence 1 in the paragraph begins "The Analysis Tools package..." and then sentence 3 begins "Analysis tools...", it's too much repetition of the phrase in quick succession. This is why I initially preferred "In that sense...", as it's clear were still discussing in the context of sentence 1.

Reply:
Use of "therefore" is an attempt to break up that repetition in an attempt to satisfy your comment.